### DISCUSSION TITLE
1Password Quick Access Window Rules

### DISCUSSION THREAD

**Discussion by heywoodlh** [hyprwm/Hyprland#2665] [Q&A]
Hey all,

I'm trying to figure out the correct way to use 1Password's Quick Access function in Hyprland. I can type for a few moments into it when it appears, but it loses focus and disappears before I can type what I needed to into it.

Here are my current Windows Rules for 1Password: 

https://github.com/heywoodlh/nixos-configs/blob/master/roles/home-manager/linux/hyprland.nix#L194-L196

Any help on this would be greatly appreciated.

**Comment by druncopsyken** [comment]
I actually had this issue with a different application. But I am also using 1 password and just added the rule. This might help you out to make it work, try adding this line by wedging it between your other 2 rules for it:
windowrulev2 = stayfocused,class:^(1Password)$
